File: backend/src/main/java/com/back/team11/domain/cafe/repository/CafeRepository.java
package com.back.team11.domain.cafe.repository;

import com.back.team11.domain.cafe.entity.Cafe;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CafeRepository extends JpaRepository<Cafe, Long>, CafeRepositoryCustom {
    boolean existsByNameAndAddress(String name, String address);
}

File: backend/src/main/java/com/back/team11/domain/cafe/repository/CafeRepository.kt
package com.back.team11.domain.cafe.repository

import com.back.team11.domain.cafe.entity.Cafe
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CafeRepository : JpaRepository<Cafe, Long>, CafeRepositoryCustom {
    fun existsByNameAndAddress(name: String, address: String): Boolean
}
